otter/convergence/composition.py

```diff
@@ -67,7 +67,7 @@ def json_to_LBConfigs(lbs_json):
         if lb.get('type') != 'RackConnectV3':
             lbd[lb['loadBalancerId']].append(CLBDescription(
                 lb_id=str(lb['loadBalancerId']), port=lb['port']))
-    return lbd
+    return dict(lbd)
```

Comment:
Yeah, the problem was that freeze(defaultdict) (which happens later) doesn't work.

I guess I can change this to turn it into persistent data structure sooner -- I'll do that.